### PR TITLE
KEYCLOAK-10757: Replaying assertion with signature in SAML adapters

### PR DESCRIPTION
**Pull request by rmartinc**
Documentation change to KEYCLOAK-10757.

### CODE REVIEW

**Comment by rmartinc** [comment]
I'm off the next two weeks, if something should be changed, feel free to do it. If not I'll do it when I come back.

**Review by sguilhen — APPROVED** [review]
Looks good to me

**Comment by sguilhen** [comment]
Just triggered another run on travis, PR seems fine now.

**Review by hmlnarik — APPROVED** [review]

File: securing_apps/topics/saml/java/general-config/sp_element.adoc [hunk 1]
```diff
@@ -62,3 +63,10 @@ logoutPage::
   the user is redirected after logout to that page using the HTTP `302` status code. If a link without scheme part is specified,
   such as `/logout.jsp`, the page is displayed after logout, _regardless of whether it lies in a protected area according
   to `security-constraint` declarations in web.xml_, and the page is resolved relative to the deployment context root.
+
+keepDOMAssertion::
+  This attribute should be set to __true__ to make the adapter store the DOM representation of the assertion in its
+  original form inside the `SamlPrincipal` associated to the request. The assertion document can be retrieved using
+  the method `getAssertionDocument` inside the principal. This is specially useful when re-playing a signed assertion.
```
Comment (andymunro) [inline securing_apps/topics/saml/java/general-config/sp_element.adoc:70]: specially should actually be "especially."   However, I would prefer "particularly".

**Pull request merged**